src/routes/login.tsx

```diff
@@ -1,6 +1,13 @@
 import Authenticated from '@/components/authenticated'
 
-const goBack = () => window.history.back()
+const useQuery = () => {
+  return new URLSearchParams(window.location.search);
+}
+
+const goBack = () => {
+  // eslint-disable-next-line caleb/react-hooks/rules-of-hooks
+  window.location.href = decodeURIComponent(useQuery().get("from") || '');
```

Comment:
I'm curious why useQuery is a separate function, does it provide any advantage here? It looks like ESLint thinks that it is a react hook (because its name starts with `use`) but as far as I can tell it's not.